src/components/SlateEditor/plugins/embed/SlateAudio.jsx

```diff
@@ -21,55 +22,92 @@ class SlateAudio extends React.Component {
     super(props);
     this.state = { audio: {} };
     this.toggleEdit = this.toggleEdit.bind(this);
+    this.onAudioFigureInputChange = this.onAudioFigureInputChange.bind(this);
   }
 
   async componentWillMount() {
```

Comment:
kanskje `const {embed: {caption, resource_id}} = this.props` isteden?

Reply:
Endret.